Makefile

```diff
@@ -81,12 +98,33 @@ distclean:
 	-rm -rf deps/icu
 	-rm -rf deps/icu4c*.tgz deps/icu4c*.zip deps/icu-tmp
 	-rm -f $(BINARYTAR).* $(TARBALL).*
+	-rm -rf deps/v8/testing/gmock
+	-rm -rf deps/v8/testing/gtest
 
 check: test
 
 cctest: all
 	@out/$(BUILDTYPE)/$@
 
+v8:
+	tools/make-v8.sh v8
+	#cd deps/v8
+ifneq (,$(filter $(DESTCPU),x86))
+	+make -C deps/v8 $(V8_BUILD_NO_I18N);
+else
+ifneq (,$(filter $(ARCH),x86))
+	+make -C deps/v8 $(V8_BUILD_NO_I18N);
+else
+ifeq ($(ARCH)$(DESTCPU),)
+	+make -C deps/v8 $(V8_BUILD_NO_I18N);
+else
+	+make -C deps/v8 $(ARCH) $(V8_BUILD_NO_I18N);
+endif
+	+make -C deps/v8 $(ARCH) $(V8_BUILD_NO_I18N);
+endif
+	+make -C deps/v8 $(ARCH) $(V8_BUILD_NO_I18N);
```

Comment:
Won't this run the same make command twice here?

Style nits: the semicolons are not necessary and it's customary to write `$(MAKE)` instead of `make` inside a Makefile.